applications/IgaApplication/custom_modelers/nurbs_geometry_modeler_sbm.cpp

```diff
@@ -0,0 +1,149 @@
+//    |  /           |
+//    ' /   __| _` | __|  _ \   __|
+//    . \  |   (   | |   (   |\__ `
+//   _|\_\_|  \__,_|\__|\___/ ____/
+//                   Multi-Physics
+//
+//  License:         BSD License
+//                   Kratos default license: kratos/license.txt
+//
+//  Main authors:    Nicolo' Antonelli
+//                   Andrea Gorgi
+//
+
+// Project includes
+#include "nurbs_geometry_modeler_sbm.h"
+#include "custom_utilities/create_breps_sbm_utilities.h"
+#include "custom_processes/snake_sbm_process.h"
+#include "iga_application_variables.h"
+
+namespace Kratos
+{
+
+    ///@name Stages
+    ///@{
+
+    void NurbsGeometryModelerSbm::SetupGeometryModel(){
+
+        //----------------------------------------------------------------------------------------------------------------
+        KRATOS_INFO_IF("NurbsGeometryModelerSbm", mEchoLevel > 1) << "[NURBS MODELER SBM]:: calling NurbsGeometryModeler" << std::endl;
```

Comment:
```suggestion
        KRATOS_INFO_IF("NurbsGeometryModelerSbm", mEchoLevel > 1) << "Calling NurbsGeometryModeler" << std::endl;
```
The macro already prints the label (in this case `NurbsGeometryModelerSbm`).